### PR TITLE
Song controller & logincontroller refactored

### PR DESCRIPTION
**Pull request by miguelgimenezgimenez**

### CODE REVIEW

File: controllers/raccoonController.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+const raccoon = require('raccoon');
+raccoon.config.className = 'spotamovie';
+
+raccoon.connect(6379, '127.0.0.1');
```
Comment (arol) [inline controllers/raccoonController.js]: No changes required, but keep in mind that this is not suitable for production.

File: controllers/movieController.js [hunk 2]
```diff
@@ -1,3 +1,29 @@
...
+const raccoonController = require('./raccoonController');
+const localAuth = require('../auth/local');
+
+
+
 let movieController = {};
+
+
+movieController.like=(req,res)=>{
+console.log(req.headers);
```
Comment (arol) [inline controllers/movieController.js]: - indent it correclty
- remove logs

**Comment by arol** [comment]
Remove encoding code.

**Comment by miguelgimenezgimenez** [comment]
removed all encryption files

**Pull request merged**